### PR TITLE
Created update:school_logos and helper schools:populate rake cmds

### LINKED ISSUE #284: Point to AWS API for Code Schools image src's
# Feature
## Why is this feature being added?

We're moving off of GitHub's assets src as an API.

## What should your feature do?
I'm not sure where the data is grabbed from anymore, but the model for code_schools includes a `logo` key. The fields should be adjusted to the format of this example url: `https://s3.amazonaws.com/operationcode-assets/codeSchoolLogos/bloc.jpg`

## Existing Data

[Here is a sorted list](https://gist.github.com/hpjaj/ef81f7409f8daff7b563a2e874693f14) of the `CodeSchool` db records in production as of March 27th.  Namely their `name` and `url` attributes:

https://gist.github.com/hpjaj/ef81f7409f8daff7b563a2e874693f14

## Definition of done

- [ ] A rake task that updates all of the `CodeSchool` records' `url` attributes accordingly
- [ ] Smoke test that the updates are working, by [running the front and backends together](https://github.com/OperationCode/operationcode_backend/wiki/Running-The-Front-and-Backend-Repos-Together) in your local dev env

### PR DESCRIPTION
**Pull request by jvillama**
# Description of changes

Creates rake tasks to update `CodeSchool` records' `logo` attribute to use aws hosted images instead of GitHub's assets.

# Issue Resolved

Fixes #284

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,11 +47,22 @@ db_rollback:
 db_seed:
 	docker-compose run ${RAILS_CONTAINER} rake db:seed
 
+.PHONY: schools_populate
```
Comment (hpjaj) [inline Makefile]: Pls revert all of these changes to the `Makefile`.  Reason being, they are one-off use cases that do not apply to recurring development.

File: test/lib/update_school_logos_test.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    end
+  end
+
+  test 'it asserts schools\' logos contain aws urls' do
+    Rake::Task['update:school_logos'].invoke
+    schools = CodeSchool.all
+    assert_kind_of CodeSchool::ActiveRecord_Relation, schools
+    schools.each do |school|
+      logo = school.logo
+      assert_match "https://s3.amazonaws.com/operationcode-assets", logo
```
Comment (hpjaj) [inline test/lib/update_school_logos_test.rb]: 👍

**Comment by hpjaj** [comment]
@jvillama - Have you [ran the front and backends together](https://github.com/OperationCode/operationcode_backend/wiki/Running-The-Front-and-Backend-Repos-Together), to smoke test that the logos are appearing correctly?

**Comment by jvillama** [comment]
Thanks 👍, @kylemh will be updating the remaining frontend references to github assets.

**Pull request merged**